src/interfacers/EmonHubDataLoggerInterfacer.py

```diff
@@ -0,0 +1,194 @@
+import time
+import serial
+import Cargo
+import string
+from pydispatch import dispatcher
+import emonhub_interfacer as ehi
+from emonhub_interfacer import EmonHubInterfacerInitError
+
+"""class EmonhubSerialInterfacer
```

Comment:
Lets add a comment that this is for the MSrX charge controler

Reply:
Done